.rgignore

```diff
@@ -0,0 +1,3 @@
+.yarn/*
```

Comment:
This file is to help searching the project with [rg (ripgrep)](https://github.com/BurntSushi/ripgrep)